include/swift/AST/DiagnosticsSema.def

```diff
@@ -8171,6 +8171,9 @@ ERROR(lifetime_dependence_invalid_inherit_escapable_type, none,
 ERROR(lifetime_dependence_cannot_use_parsed_borrow_consuming, none,
       "invalid use of borrow dependence with consuming ownership",
       ())
+ERROR(lifetime_dependence_cannot_use_default_escapable_consuming, none,
+      "invalid lifetime dependence on Escapable value with %0 ownership",
```

Comment:
to be consistent with using full grammar:
`dependence on an Escapable value`

Reply:
Done!